src/Loader.ts

```diff
@@ -24,6 +24,12 @@ interface AppManifest<T> {
     services?: ServiceDeclaration<T>[];
 }
 
+interface ProviderManifest<T> {
+    uuid: string;
```

Comment:
What is the case that you get a manifest with the uuid on the top level?

Reply:
Honestly, not sure. I lifted this from `AppManifest` above. Does a top-level UUID not actually do anything then?

Reply:
I dont think so, but there could be some nonstandard like platform or external app or something that Im missing. Maybe leave it for safety, I was just curious